src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let cnt = 0;
+
+  const getSum = (x) => (y) => {
+    cnt++;
+
+    if (cnt > 3 && cnt % 2 === 0) {
+      return 'Bzzz... Error!';
+    }
+
+    return x + y;
+  };
```

Comment:
This syntax is a bit confusing, you can do it like that to improve readability
```suggestion
  const getSum = (x) => (
    (y) => {
      cnt++;

      if (cnt > 3 && cnt % 2 === 0) {
        return 'Bzzz... Error!';
      }

      return x + y;
    }
  );
```